google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -679,8 +679,7 @@ def _convert_iam_policy(self, iam_policy):
         """
         parent, full_res_name = self._get_parent(iam_policy)
         iam_policy_type_name = to_type_name(
-            iam_policy.get_type_class(),
-            parent.type_name)
+            iam_policy.get_type_class(), parent.name)
```

Comment:
This will cause duplicate error when building the data model:
https://github.com/GoogleCloudPlatform/forseti-security/issues/1106
This will just reverse the previous fix on the above issue:
https://github.com/GoogleCloudPlatform/forseti-security/pull/1102/files#diff-5129d89f01ca9c52224878ef9ffd09e5R675